src/styles/styles.js

```diff
@@ -231,7 +231,6 @@ const styles = {
 
     badge: {
         backgroundColor: themeColors.badgeDefaultBG,
-        borderRadius: 14,
```

Comment:
For the IOUbadge, this border-radius is now set to 12 instead of 14 since I wanted to keep the increments of small/medium/large as multiples of 4 like we do for the spacing file. @shawnborton let me know if this should be set back to 14 and I can update the value in the new borders.js file.

Reply:
I think for badges, we want them to look like rounded pills and not rounded rectangles because buttons look like rounded rectangles. So maybe go the other way and jack this up to 16 or 20?

Reply:
Here's what 16 and 20 look like, I can update to either
| 16 | 20 |
|---|---|
| ![image](https://user-images.githubusercontent.com/3981102/115638651-a5df5280-a2c7-11eb-94c2-e023ef95cb49.png) | ![image](https://user-images.githubusercontent.com/3981102/115638705-c0b1c700-a2c7-11eb-94e0-9873513a41bb.png) |

Reply:
Hmm I am not seeing a difference. Just to make sure we're on the same page, I think the style we are using for the `$2.00` in your mockup is the general style we have for badges, and we should just reuse that style for the Chats header too.

Reply:
Ah ok sounds good, I'll revert the styles then so both badges have the old border-radius that the IOUBadge had

Reply:
Just for confirmation, how does this look?
![image](https://user-images.githubusercontent.com/3981102/115752580-167d8200-a34f-11eb-86a8-e8ec92614308.png)

Reply:
For some reason the pill doesn't look perfectly rounded, but maybe my eyes are playing tricks on me. Can you double check that the "DEV" pill is only 20px tall? if that's the case, a border radius of > 10px should make it perfectly round.

Reply:
Here's the box model and element style for the DEV div
![image](https://user-images.githubusercontent.com/3981102/115754337-f2bb3b80-a350-11eb-950d-41fc4d8a1178.png) ![image](https://user-images.githubusercontent.com/3981102/115754527-2007e980-a351-11eb-9c63-fc87fdcd4880.png)

Reply:
Is it just me, or is the text in the pill (both `DEV` and `$2.00`) not completely centered?

Reply:
Can we see w/ `borderRadius: 14`?

Reply:
Here it is with 14
![image](https://user-images.githubusercontent.com/3981102/115775591-ef7f7a00-a367-11eb-82f5-40662c4a0a2d.png)